account_payment_return/models/account_invoice.py

```diff
@@ -31,3 +31,14 @@ class AccountInvoice(models.Model):
         string='Payment returned',
         help='Invoice has been included on a payment that has been returned '
              'later.')
+    bank_charge = fields.Float(
```

Comment:
It should be called "bank_fee" (but you can put oldname="bank_charge" to keep your compatibility)